src/components/HeroSection/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Button from '../Shared/Buttons';
+const Hero = () => {
+  return (
+    <section className="heroSection">
+      <div>
+        <p className="paragraph--p2 hero__subheading">AI-powered chatting PLATFORM</p>
+        <h1 className="heading--h1">Learning and teaching online, made easy.</h1>
+        <p className="paragraph--p1">
+          Practice your English and learn new things with the platform.
+        </p>
+        <Button color="light" label="About Platform" />
+
+        <div className="hero__figure">
+          <div className="hero__figure_left">
+            <h3 className="heading--h3">
+              <img src="/heroLightening.svg" alt="heroLightening icon" />
+              600<span>+</span>
+            </h3>
```

Comment:
1. it is technically possible to nest an `<img>` tag within an `<h3>` tag, but it is not considered semantically correct, so please keep the `<img>` tag outside of the `<h3>` tag and use
2. you should change `letter-spacing` to `-0.02em`